sample/README.md

```diff
@@ -0,0 +1,45 @@
+## Sample app
+
+### Generate Dependency Information
+
+```bash
+./gradlew :sample:shared:exportLibraryDefinitions -PaboutLibraries.exportPath=src/commonMain/composeResources/files/
+```
+
+### Generate reference images for tests
+
+```
```

Comment:
can you please make all of these also `bash`

Reply:
Thanks for pointing it out